### PR TITLE
feat: Create as command in Fluent 25.1

### PR DESCRIPTION
**Pull request by mkundu1**
`create()` is available as a command in Fluent 25.1. This PR contains PyFluent side change.

`CreatableNamedObjectMixinOld` - mixin with create method for Fluent < 25.1.
`CreatableNamedObjectMixin` - mixin without create method for Fluent >= 25.1 as create is available as a command.

Please see the changed test_wildcard_fnmatch for the currently supported usage (backward compatibility is provided).

### CODE REVIEW

File: src/ansys/fluent/core/solver/flobject.py [hunk 1]
```diff
@@ -1747,8 +1747,22 @@ def __len__(self):
 
 
 class CreatableNamedObjectMixin(collections.abc.MutableMapping, Generic[ChildTypeT]):
-    """Provides creatable named objects."""
+    """Provides creatable named objects for Fluent 25.1 and later."""
```
Comment (PipKat) [inline src/ansys/fluent/core/solver/flobject.py:1750]: For our user-facing docs, we are supposed to use this version convention: Fluent 2025 R1.

**Review by seanpearsonuk — APPROVED** [review]

File: src/ansys/fluent/core/solver/flobject.py [hunk 2]
```diff
@@ -1749,8 +1751,22 @@ def __len__(self):
...
+        child = self._objects.get(name)
+        if not child:
+            child = self._create_child_object(name)
+        child.set_state(value)
+
+
+class CreatableNamedObjectMixinOld(CreatableNamedObjectMixin):
+    """Provides creatable named objects for Fluent 24.2 and earlier."""
+
+    # In Fluent 25.1, create method is available as commands in NamedObject class.
```
Comment (PipKat) [inline src/ansys/fluent/core/solver/flobject.py:1769]:
```suggestion
    """Provides creatable named objects for Fluent 2024 R2 and earlier."""

    # In Fluent 2025 R1, the ``create()`` method is available as commands in the ``NamedObject`` class.
```
User-facing doc should format Ansys versions like this: 2024 R2. In the comment, the edits to other than the release use the double backticks to surround code entites.

Comment (mkundu1) [reply src/ansys/fluent/core/solver/flobject.py:1769]: Actually, this is a code comment. It won't be visible in the user-facing doc.

Comment (PipKat) [reply src/ansys/fluent/core/solver/flobject.py:1769]: True, but the previous class description will be. It is best to keep the terminology consistent!

**Review by prmukherj — APPROVED** [review]

**Pull request merged**